templates/meta-time.php

```diff
@@ -26,7 +26,7 @@
 			sprintf(
 				/* translators: %s: the human-readable time difference. */
 				__( '%s ago', 'amp' ),
-				human_time_diff( $this->get( 'post_publish_timestamp' ), time() )
+				human_time_diff( $this->get( 'post_publish_timestamp' ), current_time('timestamp') )
```

Comment:
This actually used to be `current_time( 'timestamp' )` before, but it was changed in https://github.com/ampproject/amp-wp/pull/3710/commits/39473e393dba20af83136f4fef8fa4eb70db58f2. Its use was discouraged as of https://github.com/WordPress/WordPress-Coding-Standards/issues/1791.